src/qnngds/cells.py

```diff
@@ -560,13 +560,16 @@ def ntron(
     return DIE_NTRON
 
 
-def snspd(
+def snspds(
     die_w: Union[int, float] = dflt.die_w,
     pad_size: Tuple[float] = dflt.pad_size,
-    snspd_width: float = 0.2,
-    snspd_pitch: float = 0.6,
+    snspds_width_pitch: List[Tuple[float, float]] = [
+        (0.1, 0.3),
+        (0.2, 0.6),
+        (0.3, 0.9),
+    ],
     snspd_size: Tuple[Union[int, float], Union[int, float]] = tuple(
-        x / 2 for x in utility.calculate_available_space_for_dev()
+        x / 3 for x in utility.calculate_available_space_for_dev()
```

Comment:
This value is not an integer (`126.66666666666667`), maybe try to round it to a decent precision so that it's not so long on the qnngds API

Reply:
In the function description you should add how the User can create X SNSPDs.
Something like "_The function creates as many devices as there are `snspds_width_pitch` tuples_".

Reply:
Also, `3` is a magic number, maybe use the size of `snspds_width_pitch` instead?